app/controllers/stafftools_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StafftoolsController < ApplicationController
```

Comment:
Yeah, it's a personal-preference thing, but I definitely like seeing authorization checks at the controller level.